src/views/LoginView.vue

```diff
@@ -184,47 +285,116 @@ export default {
 </script>
 
 <style scoped lang="scss">
+
+.loginViewContainer{
+  background-color: #fff4d7;
+}
+
+.welcomeContainer{
+  min-width: 100vw;
+  min-height: 100vh;
+  text-align: center;
+  color: #FFF4D7;
+  background: center / cover no-repeat  url('img/game/board-background.svg');
+  box-shadow: inset 0 0 700px -1px #000000;
+}
+
+.cardLogo{
+  min-height: 80px;
+  height: 9vh;
+}
+
+.welcomeContainer h1{
+  font-family: 'Luckiest Guy', serif !important;
+  font-size: 5rem;
+  letter-spacing:.2rem;
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.video-container__wrapper {
+  width: 50%;
+  max-width: 650px;
+  margin: 0 auto;
+}
+
+/* https://css-tricks.com/fluid-width-video/ */
+.video-container {
+  position: relative;
+  padding: 0 0 56.25%;
+  height: 0;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.video-container__video {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border: none;
+}
+
+
 .container {
   width: 75%;
-  margin: 10px auto;
   display: flex;
   justify-content: center;
   flex-direction: column;
 }
 
+form{
+  max-width: 540px;
+}
```

Comment:
nitpick but let's give the form a class. It's generally more efficient for the DOM to render css by class than by tag selectors